services/common/src/components/common/LeafletMap.tsx

```diff
@@ -134,7 +134,7 @@ const LeafletMap: FC<LeafletMapProps> = ({ mine, additionalPins = [], controls =
           pin.remove();
         }
       });
-      createAdditionalPins(additionalPins);
+      if (layerGroup) createAdditionalPins(additionalPins);
```

Comment:
I'm wondering if it's going to run the useEffect again when the layerGroup _does_ exist or if it'll stay blank (or not place the pins I guess). Should layerGroup be included in the dep array?

Reply:
Yeah, nice catch. I have added it layerGroup to the dependency array int he useEffect.